docs/admin-manual/configuration-macros.rst

```diff
@@ -8085,8 +8085,7 @@ condor_procd Configuration File Macros
     own. Use of the :tool:`condor_procd` results in improved scalability
     because only one instance of this logic is required. The
     :tool:`condor_procd` is required when using group ID-based process
-    tracking (see :ref:`admin-manual/ep-policy-configuration:group
-    id-based process tracking`.
+    tracking.
```

Comment:
Let's start to remove all references to "group ID-based process tracking" from the manual.  I don't think anyone uses it, and I'd like to remove the code eventually.